src/libsyncengine/jobs/local/localcopyjob.cpp

```diff
@@ -37,6 +37,12 @@ bool LocalCopyJob::canRun() {
     if (!IoHelper::checkIfPathExists(_dest, exists, ioError)) {
```

Comment:
IoHelper::checkIfPathExists returns/should return false only for unexpected errors (like all app functions).
ExitCode::SystemError & ExitCause::FileAccessError is an expected error.

Reply:
Here, I check the returned value of `checkIfPathExists`. If it is `false`, we then exit with:
```cpp
_exitCode = ExitCode::SystemError;
_exitCause = ExitCause::Unknown;
```
If the return value is `true`, I then check `if (ioError == IoError::AccessDenied)` and, if needed, return:
```cpp
_exitCode = ExitCode::SystemError;
_exitCause = ExitCause::FileAccessError;
```
In this case, I return `false`, as we are in `bool LocalCopyJob::canRun()`, where the return value indicates if the job can run, which is not the case if we do not have sufficient permissions.